mlir/utils/jenkins/Jenkinsfile

```diff
@@ -546,25 +546,25 @@ pipeline {
                             dir('build') {
                                 timeout(time: 60, activity: true, unit: 'MINUTES') {
                                     // Tune gemms, fail if the DB is not created
-                                    sh """python3 ./bin/tuningRunner.py --operation gemm \
-                                            --configs_file=../mlir/utils/jenkins/ci-configs/selected-gemm-configs \
-                                            --output=tuning_gemm.tsv
+                                    sh """../mlir/utils/tuna/tuna-script.sh -o gemm \
```

Comment:
Replace invocation of tuningRunner.py with use of tuna-script.sh, which will set up for Tuna and then do the same tuning.